src/php/types/object.rs

```diff
@@ -161,6 +164,38 @@ impl ZendObject {
     pub(crate) fn refcount_inc(&mut self) {
         self.gc.refcount += 1;
     }
+
+    pub(crate) fn to_string(&self) -> String {
+        let mut ret = Zval::new();
+        unsafe {
+            zend_call_known_function(
+                (*self.ce).__tostring,
+                self as *const _ as *mut _,
+                self.ce,
+                &mut ret,
+                0,
+                ptr::null_mut(),
+                ptr::null_mut(),
+            );
+        }
+
+        if let Some(err) = ExecutorGlobals::take_exception() {
+            let class_name = unsafe { ZendString::from_ptr((*self.ce).name, false) };
+            panic!(
+                "Uncaught exception during call to {}::__toString(): {:?}",
+                class_name.unwrap().as_str().unwrap(),
```

Comment:
These `unwrap()` errors are OK if you are panicing, will just need to disable the lint for the function.

Reply:
If you want to keep the lint, I can change them to expect. I don't intend for these to stay panics forever. PHP recovers from both situations. I'll put in comments when I revisit this.

But in order to remove the panics, first I need a sensible way to turn a Zval into an Error.